fed/proxy/barriers.py

```diff
@@ -407,21 +271,20 @@ def start_send_proxy(
     party: str,
     logging_level: str,
     tls_config: Dict = None,
-    retry_policy=None,
-    max_retries=None,
-    actor_config: Optional[fed_config.ProxyActorConfig] = None
+    proxy_cls=None,
+    proxy_config: Optional[fed_config.CrossSiloCommConfig] = None
 ):
     # Create SendProxyActor
     global _SEND_PROXY_ACTOR
```

Comment:
Can you add some comments why we need  a global `_SEND_PROXY_ACTOR`. (to prevent gc)

Reply:
Better do it in another PR ~